test/functional/apps/dashboard/_dashboard.js

```diff
@@ -86,42 +80,43 @@ bdd.describe('dashboard tab', function describeIndexTests() {
         });
     });
   });
+});
 
 
-  bdd.it('filters when a pie chart slice is clicked', async function () {
-    let descriptions = await PageObjects.dashboard.getFilterDescriptions(1000);
-    expect(descriptions.length).to.equal(0);
-
-    await PageObjects.dashboard.filterOnPieSlice();
-    descriptions = await PageObjects.dashboard.getFilterDescriptions();
-    expect(descriptions.length).to.equal(1);
-  });
+bdd.it('filters when a pie chart slice is clicked', async function () {
+  let descriptions = await PageObjects.dashboard.getFilterDescriptions(1000);
+  expect(descriptions.length).to.equal(0);
 
-  bdd.it('retains dark theme in state', async function () {
-    await PageObjects.dashboard.useDarkTheme(true);
-    await PageObjects.header.clickVisualize();
-    await PageObjects.header.clickDashboard();
-    const isDarkThemeOn = await PageObjects.dashboard.isDarkThemeOn();
-    expect(isDarkThemeOn).to.equal(true);
-  });
+  await PageObjects.dashboard.filterOnPieSlice();
+  descriptions = await PageObjects.dashboard.getFilterDescriptions();
+  expect(descriptions.length).to.equal(1);
+});
```

Comment:
It might be best to split this into two tests, each with a single assertion. Otherwise, if one of them fails, it might be a little difficult to tell which one is failing.

```javascript

bdd.describe('filters', () => {
  bdd.it('are not selected, by default', async function () {
    let descriptions = await PageObjects.dashboard.getFilterDescriptions(1000);
    expect(descriptions.length).to.equal(0);
  });

  bdd.it('are added when a pie chart slice is clicked', async function () {
    let descriptions = await PageObjects.dashboard.getFilterDescriptions(1000);
    await PageObjects.dashboard.filterOnPieSlice();
    descriptions = await PageObjects.dashboard.getFilterDescriptions();
    expect(descriptions.length).to.equal(1);
  });
});
```